Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS.py

```diff
@@ -0,0 +1,1095 @@
+from typing import Dict, Tuple, List
+
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import urllib3
+import traceback
+from dateparser import parse
+
+# Disable insecure warnings
+
+urllib3.disable_warnings()
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+def incident_priority_to_dbot_score(score: int):
+    """Converts the CyberArk score to DBot score representation,
+    while the CyberArk score is a value between 0.0-100.0 and the DBot score is 1,2 or 3.
+    Can be one of:
+        0.0 - 35.0 ->  1
+        35.1 - 75.0 ->  2
+        75.1 - 100.0 ->  3
+    """
+
+    if 0 <= score <= 35:
+        return 1
+    if 35 < score <= 75:
+        return 2
+    if 75 < score <= 100:
+        return 3
+    demisto.info(f'CyberArk PAS incident score: {score} is not known. Setting as unknown(DBotScore of 0).')
+    return 0
```

Comment:
if the arg is of type int, it means it is always exist.
this case implies otherwise -> Any[int]
is it the case? if not, rm these lines.